c/test/CMakeLists.txt

```diff
@@ -6,34 +6,35 @@ if (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 endif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
 
 FILE(GLOB generated_c_sources auto*.c)
-add_executable(test_libsbp check_main.c check_edc.c check_sbp.c ${generated_c_sources})
-
-target_link_libraries(test_libsbp PRIVATE ${TEST_LIBS})
-set_target_properties(test_libsbp PROPERTIES
-        C_STANDARD 99
-        C_STANDARD_REQUIRED ON)
 
 if(MSVC)
-  target_include_directories(test_libsbp PRIVATE ${PROJECT_SOURCE_DIR}/include/libsbp/)
+  set(TEST_INCLUDES ${PROJECT_SOURCE_DIR}/include/libsbp/)
 else()
-  target_include_directories(test_libsbp PRIVATE ${PROJECT_SOURCE_DIR}/include/libsbp/)
+  set(TEST_INCLUDES ${PROJECT_SOURCE_DIR}/include/libsbp/)
   if(APPLE)
     # Some libraries are available in non-standard places on apple.
-    target_include_directories(test_libsbp PRIVATE /usr/local/include)
+    set(test_libsbp /usr/local/include)
```

Comment:
`test_libsbp` isn't being used elsewhere that I can tell

Reply:
Yep, that should be updating `TEST_INCLUDES`